### DISCUSSION TITLE
We should show the current Location

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#4011] [Ideas]
It would be nice if the Replay viewer showed the current location.href as well as the screenshot. This will help visualize when the location changes because of a push state or navigation event.

**Comment by jonbell-lot23** [comment]
Have we done this? We have this now:

<img width="241" alt="image" src="https://user-images.githubusercontent.com/9154902/158481856-683b8f73-61de-4623-9035-adf4e8376347.png">

If not, can you describe where you're thinking we can do this?
